### PR TITLE
fix: activate all interaction on edit mode at WidgetFrame and remove unnecessary groupBy from DataTransferWidget

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Commit messages say it all, so please review in order by commits.
This PR upgrade the version of vitest.

### CODE REVIEW

File: apps/web/vite.config.js [hunk 1]
```diff
@@ -47,6 +47,9 @@ export default defineConfig(async ({ command, mode }) => {
             globals: true,
             environment: 'jsdom',
             include: ['./src/**/__tests__/**/*.+(ts|js)'],
+            coverage: {
+                provider: 'istanbul',
+            },
```
Comment (sulmoJ) [inline apps/web/vite.config.js:52]: Question: What is it??

**Review by piggggggggy — APPROVED** [review]
LGTM!
Please share about `Coverage` haha..

**Review by sulmoJ — APPROVED** [review]
LGTM~!
I'm really looking forward to the test coverage results! Let's merge it quickly and check 🤓 🤓

File: apps/web/vite.config.js (hunk 1 above)
Comment (sulmoJ) [reply apps/web/vite.config.js:52]: Completed understanding

**Pull request merged**